test/libwhisper/bloomFilter.cpp

```diff
@@ -244,4 +244,53 @@ BOOST_AUTO_TEST_CASE(bloomFilterRaw)
 	BOOST_REQUIRE(!f.contains(b00110111));
 }
 
-BOOST_AUTO_TEST_SUITE_END()
\ No newline at end of file
+enum { DistributionTestSize = 8 };
```

Comment:
`static const int DistributionTestSize = 8;` is preferred as it doesn't break GCC.